kdesk/icon-hooks.sh

```diff
@@ -102,30 +102,29 @@ case $icon_name in
 
     "world")
         IFS=$'\n'
-        kano_statuses=`kano-profile-cli get_notifications_count`
-        is_online=`kano-profile-cli is_registered`
+        # Returns if online, prints notification count and stats activity
+        info=`kano-profile-cli get_world_info`
         apirc=$?
 
         if [ "$debug" == "true" ]; then
-            printf "Kano Profile API returns rc=$apirc, data=\n$kano_statuses\n"
+            printf "Kano Profile API returns rc=$apirc, data=\n$info\n"
```

Comment:
I know it is a bit pedantic but using https://github.com/KanoComputing/kano-toolset/blob/master/bash/logging.sh for logging would be better because we would then be able to access the logs through `kano-logs`.

Example of use:
* First source it : https://github.com/KanoComputing/kano-desktop/blob/master/bin/kano-uixinit#L19
* Then log stuff: https://github.com/KanoComputing/kano-desktop/blob/master/bin/kano-uixinit#L50